lua/go/codelens.lua

```diff
@@ -56,7 +56,7 @@ function M.refresh()
     vim.lsp.codelens.refresh({ bufnr = 0 })
   else
     log('refresh codelens')
-    vim.lsp.codelens.clear()
+    vim.lsp.codelens.clear(gopls.id, 0)
```

Comment:
maybe apply the same changes to line :41 as well

Reply:
sure